spec/cloudflair/api/zone/settings/development_mode_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 
 describe Cloudflair::DevelopmentMode do
+  subject(:development_mode) { Cloudflair.zone(zone_identifier).settings.public_send setting_identifier }
```

Comment:
mhm... then let's leave it like it is and I search for a solution when I do the refactoring.